framework/components/ecl-messages/ecl-messages.twig

```diff
@@ -1,5 +1,15 @@
-<div class="messages alert {{messages_type}}">
-  <a href="#" class="close" data-dismiss="alert">&times;</a>
-  <h3><span class="ecl-sr-only">{{messages_sr_only}}:</span>{{messages_title}}</h3>
-  <p>{{messages_data}}</p>
+<div class="ecl-message {% if modifier is defined %} ecl-message--{{ modifier }} {% endif %}">
```

Comment:
Not to my knowledge. The only dialog I know of is the language switcher on inner pages. Galleries maybe also, but we have to double-check them when time comes.
These are the messages shown in Drupal, as far as I know.